qa/rpc-tests/test_framework/cashlib/cashlib.py

```diff
@@ -286,6 +286,7 @@ class ScriptFlags(IntFlag):
     SCRIPT_ENABLE_CHECKDATASIG = (1 << 18)
     SCRIPT_DISALLOW_SEGWIT_RECOVERY = (1 << 20)
     SCRIPT_ENABLE_MUL_SHIFT_INVERT = (1 << 21)
+    SCRIPT_ENABLE_OP_REVERSEBYTES = (1 << 23)
 
```

Comment:
why 1<<23 and not 1<<22  for SCRIPT_ENABLE_OP_REVERSEBYTES?

Reply:
1U << 22 is reserved for SCRIPT_VERIFY_INPUT_SIGCHECKS (the other feature that will be added in the next update)